src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivityResourceIntTest.kt

```diff
@@ -246,6 +253,29 @@ class ActivityResourceIntTest : IntegrationTestBase() {
       }
     }
 
+    @Test
+    fun `should raise telemetry event`() {
+      val id = callCreateEndpoint()
+      val courseActivity = repository.lookupActivity(id)
+
+      verify(telemetryClient).trackEvent(
+        eq("activity-created"),
+        check<MutableMap<String, String>> { actual ->
+          mapOf(
+            "courseActivityId" to id.toString(),
+            "prisonId" to PRISON_ID,
+            "courseScheduleIds" to "[${courseActivity.courseSchedules[0].courseScheduleId}]",
+            "courseActivityPayRateIds" to "[BAS-5-2022-10-31]",
+            "courseScheduleRuleIds" to "[${courseActivity.courseScheduleRules[0].id}]",
+          ).also { expected ->
+            log.info("expected telemetry details to be: $expected")
```

Comment:
This test was failing but not printing the expected values! So I'm logging them manually 🤷